Fixed #32647 -- Restored multi-row select with shift-modifier in admin changelist. (#14288)
django/django: docs/releases/3.2.1.txt

* Fixed a regression in Django 3.2 that caused a crash when decoding a cookie
  value, used by ``django.contrib.messages.storage.cookie.CookieStorage``, in
  the pre-Django 3.2 format (:ticket:`32643`).

> * Fixed a regression in Django 3.2 when using the shift-key modifier to select

Reviewer: Maybe:
```suggestion
* Fixed a regression in Django 3.2 that stopped the shift-key modifier to select
```
